guides/source/configuring.md

```diff
@@ -465,7 +465,7 @@ encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
   method should be performed on the parameters. See [Security Guide](security.html#unsafe-query-generation)
   for more information. It defaults to `true`.
 
-* `config.action_dispatch.rescue_responses` configures what exceptions are assigned to an HTTP status. It accepts a hash and you can specify pairs of exception/status. By default, this is defined as:
+* `config.action_dispatch.rescue_responses` configures what exceptions are assigned to an HTTP status. It accepts a hash and you can specify pairs of exception/status. Please note that this is only for the purposes of logging. No exceptions are actually rescued. By default, this is defined as:
```

Comment:
Apologies, my mistake. I made some wrong assumptions that led me up the garden path! I blame the man flu from which I have been recovering.

I now see that the exception is rescued from within `ActionPack::ShowExceptions`.